creator-node/src/fileManager.js

```diff
@@ -350,7 +350,7 @@ const trackFileUpload = multer({
       cb(null, true)
     } else {
       req.fileFilterError = `File type not accepted. Must be one of [${ALLOWED_UPLOAD_FILE_EXTENSIONS}] with mime type matching ${AUDIO_MIME_TYPE_REGEX}, got file ${fileExtension} with mime ${file.mimetype}`
-      cb(null, false)
+      cb(req.fileFilterError, false)
```

Comment:
i think this should be fine? unfortunately the two people who would know this best are @SidSethi and @raymondjacobson and theyre both out this week

Reply:
Sure, I can revert this for the time being, and we can come back to this again.